vhdbuilder/cgmanifest.json

```diff
@@ -10,6 +10,17 @@
                 }
             },
             "DevelopmentDependency": false
+        },
```

Comment:
The file name is "cgmanifest.json" and you can have as many as you need and can be anywhere in your repository.